src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -14,6 +18,10 @@
 
     private static final String FXML = "EventListCard.fxml";
 
+    private static final String MESSAGE_PERSON_LIMIT_REACHED = "... %d more attendee(s)";
+
+    private static final int associatedPersonsLimit = 6;
```

Comment:
is there a particular reason why the max number of persons to be displayed is 6?

Reply:
No particular reason, it just looked nice in GUI I guess. I'll leave a TODO message to maybe shift this out to be included under UserPrefs.